src/cli/helpers.ts

```diff
@@ -195,7 +195,8 @@ function emitTestCaseHooks(
   ;[]
     .concat(
       supportCodeLibrary.beforeTestCaseHookDefinitions,
-      supportCodeLibrary.afterTestCaseHookDefinitions
+      // order has been reversed before - need it back in original order
```

Comment:
Can we solve this in a different way by having the support code library keep it in the original order and only reverse it in the runtime when we are about to execute them?

Reply:
done